.evergreen/install-cli.sh

```diff
@@ -10,6 +10,7 @@ fi
 
 if [ -z "${1:-}" ]; then
   echo "Must give a target directory!"
+  exit 1
```

Comment:
Consider the following instead of `-z` + `exit`:

```bash
: "${1:?"must give a target directory!"}"
```

Reply:
Done